<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginAccount</class>
 <widget class="QWidget" name="LoginAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginAccount</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:30px;
color:white;</string>
   </property>
   <property name="text">
    <string>Employee Login</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>441</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>38</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font-size:20px;
color:white;</string>
        </property>
        <property name="text">
         <string>Enter Employee Id</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="login_emp_id"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>207</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>55</number>
    </property>
    <item>
     <widget class="QPushButton" name="login_ok_btn">
      <property name="styleSheet">
       <string notr="true">font-size:20px;
color:black;
background-color: white;</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="login_cancel_btn">
      <property name="styleSheet">
       <string notr="true">font-size:20px;
color:black;
background-color: white;</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
